README.md

```diff
@@ -67,9 +67,9 @@ Environment variables available for configuration are listed below:
 | COLLECTION_EXERCISE_PROTOCOL | Protocol used for collection exercise service uri | 'http'
 | COLLECTION_EXERCISE_HOST     | Host address used for collection exercise uri | 'localhost'
 | COLLECTION_EXERCISE_PORT     | Port used for collection exercise uri | 8145
-| RM_SURVEY_SERVICE_PROTOCOL   | Protocol used for rm survey service uri | 'http'
-| RM_SURVEY_SERVICE_HOST       | Host address used for rm survey service uri | 'localhost'
-| RM_SURVEY_SERVICE_PORT       | Port used for rm survey service uri | 8080
+| SURVEY_SERVICE_PROTOCOL   | Protocol used for rm survey service uri | 'http'
```

Comment:
Stupidly minor but could you realign the lines

Reply:
This will automatically display in table form when displayed as part of the README.md